### PR TITLE
packaging: remove python3*-devel wildcard

### PR DESCRIPTION
**Pull request by tswhison**
It was reported that the wildcard doesn't work with Fedora 37. Move
to an explicit check on the name instead.

Add libedit-devel and systemd-devel to the list of prerequisites for
the full package build.

Signed-off-by: Tim Whisonant <tim.whisonant@intel.com>

### CODE REVIEW

**Comment by tswhison** [comment]
Submitting this instead of #2646, as that one was submitted from a fork, which causes our CI to hang.

**Comment by gwencasey96** [comment]
I thought you mentioned the python3*-devel was needed for rhel 8? Sorry I didn't get a chance to confirm. Will Rhel 8 and 9 be added to this CI? Also I don't believe I had direct access so I submitted from a fork; please let me know if there's a preferred way for me to communicate changes.

**Comment by tswhison** [comment]
The addition of python36-devel and python38-devel as explicit options satisfies the RHEL builds.  Let me see about adding you and Tom as developers.

**Comment by tswhison** [comment]
@gwencasey96 I updated your account to have write access.  Hopefully that will solve it.

File: packaging/opae/rpm/create [hunk 1]
```diff
@@ -50,6 +50,27 @@ declare -r RPM_SPEC="opae.spec.${WHICH_RPM}"
 
 # Check for prerequisite packages.
 
+declare -ra PYTHON_PKG_DEPS=(\
+'python3-devel' \
+'python36-devel' \
+'python38-devel'\
```
Comment (anandaravuri) [inline packaging/opae/rpm/create:56]: missing 3.9,3.10, and 3.11 ?
Fedora37 python3.11 version
fedora36 python3.10 version

default pyhon packages changes every Fedora release
can we use python3*-devel ?

Comment (tswhison) [reply packaging/opae/rpm/create:56]: We can't use python3*-devel.  Per Gwen, it breaks on Fedora37.  What are the exact package names in Fedora36 and 37?

Comment (anandaravuri) [reply packaging/opae/rpm/create:56]: Fedora37  cat /etc/redhat-release
Fedora release 37 (Rawhide)
python
Python 3.11.0b3 (main, Jun 24 2022, 00:00:00) [GCC 12.1.1 20220507 (Red Hat 12.1.1-1)] on linux

Fedora35  cat /etc/redhat-release
Fedora release 35 (Thirty Five)
 python
Python 3.10.4 (main, Mar 25 2022, 00:00:00) [GCC 11.2.1 20220127 (Red Hat 11.2.1-9)] on linux


Python in Fedora 36, Python 3.10,

**Review by anandaravuri — APPROVED** [review]

**Pull request merged**